client/db_test.go

```diff
@@ -0,0 +1,130 @@
+package client_test
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/cockroachdb/cockroach/client"
+	"github.com/cockroachdb/cockroach/server"
+)
+
+func setup() (*server.TestServer, *client.DB) {
+	s := server.StartTestServer(nil)
+	db := client.Open("https://root@" + s.ServingAddr())
+	return s, db
+}
+
+func ExampleDBPutGet() {
+	s, db := setup()
+	defer s.Stop()
+
+	if err := db.Put("aa", "1"); err != nil {
+		log.Fatal(err)
+	}
+	result := db.Get("aa")
+	fmt.Println(result.Rows[0].String())
+
+	// Output:
+	// aa:1
+}
+
+func ExampleDBInc() {
+	s, db := setup()
+	defer s.Stop()
+
+	if r := db.Inc("aa", 100); r.Err != nil {
+		log.Fatal(r.Err)
+	}
+	result := db.Get("aa")
+	fmt.Println(result.Rows[0].String())
+
+	// Output:
+	// aa:100
+}
+
+func ExampleDBBatch() {
+	s, db := setup()
+	defer s.Stop()
+
+	b := client.B.Get("aa").Put("bb", "2")
+	if err := db.Run(b); err != nil {
+		log.Fatal(err)
+	}
+	fmt.Println(b.Results[0].Rows[0].String())
```

Comment:
just in case you didn't see them, I made some comments about this part in https://github.com/cockroachdb/cockroach/pull/871#discussion-diff-29486757